### PR TITLE
[Doc] Fixed syntax typo in CKEditor configuration

### PR DESCRIPTION
**Pull request by mountain-code**
Syntax fix for ckeditor formMapper example.

### CODE REVIEW

File: Resources/doc/reference/extra.rst [hunk 1]
```diff
@@ -148,8 +148,9 @@ Now, just create a field with ckeditor as type and your done:
                             'items' => array('Image'),
                         ),
                     )
-                );
-            );
+                )
+            )
```
Comment (core23) [inline Resources/doc/reference/extra.rst]: indention looks wrong

Comment (mountain-code) [inline Resources/doc/reference/extra.rst]: @core23 indention changed

File: Resources/doc/reference/extra.rst [hunk 2]
```diff
@@ -137,18 +137,19 @@ Now, just create a field with ckeditor as type and your done:
...
-                        ),
-                        array(
-                            'name' => 'insert',
-                            'items' => array('Image'),
-                        ),
+                    'config' => array(
+                        'toolbar' => array(
+                            array(
+                                'name' => 'links',
+                                'items' => array('Link','Unlink'),
```
Comment (OskarStark) [inline Resources/doc/reference/extra.rst]: can you please add a space after `Link',`

**Comment by core23** [comment]
Can you please squash and rebase @mountain-code

**Comment by mountain-code** [comment]
@core23 update - hope I did not went wrong

**Comment by core23** [comment]
Sorry, it did not work. Please have a look at https://github.com/sonata-project/SonataAdminBundle/blob/master/CONTRIBUTING.md#squash-your-commits

**Comment by mountain-code** [comment]
@core23 Sorry - first pull request ever on an open source project - did it again.

**Comment by greg0ire** [comment]
Please change the commit message, "squash and rebase" is probably not what you intended to use.

**Comment by mountain-code** [comment]
@greg0ire done

**Pull request closed without merging**